src/elevator.py

```diff
@@ -21,12 +20,16 @@ class ElevatorSubsystem:
     L4_POS = 45
 
     def __init__(self):
-        self.table = NetworkTableInstance.getDefault().getTable("telemetry")
+        self.table = (
+            NetworkTableInstance.getDefault()
+            .getTable("telemetry")
+            .getSubTable("Elevator Subsystem")
```

Comment:
What do you think about replacing "Elevator Subsystem" with `self.__class__.__name__`  ?? If we have multiple objects of the same type it would get messy.  I'm not sure I'm sold on this...